challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<p>I think with this I usually try to to split these into a multiple cells, and maybe include the explanation as markdown rather than comments, but that is your call as its just a stylistic preference, and if yo prefer this layout aesthetically (or funcationally in some way) thats absolutely fine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I do prefer this style for notebook settings. I think the notebook can be improved with some explaination as to why the notebook settings exist though, so I have added this.